client/src/components/ContentNavigator/ContentDataProvider.ts

```diff
@@ -334,6 +343,51 @@ class ContentDataProvider
     return success;
   }
 
+  public async handleCreationResponse(
+    resource: ContentItem,
+    newUri: Uri | undefined,
+    errorMessage: string,
+  ): Promise<void> {
+    if (!newUri) {
+      window.showErrorMessage(errorMessage);
+      return;
+    }
+
+    this.reveal(resource);
+  }
+
+  public async convertSasnbToFlw(
+    item: ContentItem,
+    name: string,
+  ): Promise<string | undefined> {
+    const parent = await this.getParent(item);
+    const resourceUri = getUri(item);
+    try {
+      // get the content of the .sasnb file
+      const contentString: string = await this.provideTextDocumentContent(
+        resourceUri,
+      );
+      // convert the .sasnb file to a flowData object
+      const flowDataUint8Array = convertSASNotebookToFlow(contentString, name);
+      // error if content is empty
```

Comment:
This comment is unneeded

Reply:
done in f5253ce014a5fec429456474725f717e0f2609a0